packages/gatsby-cli/src/create-cli.ts

```diff
@@ -513,12 +514,15 @@ export const createCli = (argv: Array<string>): yargs.Arguments => {
       describe: `Useful commands relating to Gatsby plugins`,
       builder: yargs =>
         yargs.positional(`cmd`, {
-          choices: [`docs`],
-          describe: "Valid commands include `docs`.",
+          choices: [`docs`, `verify`],
+          describe: "Valid commands include `docs`, `verify`.",
           type: `string`,
         }),
-      handler: handlerP(({ cmd }) => {
-        if (cmd === `docs`) {
+      handler: handlerP(async ({ cmd }) => {
+        if (cmd === `verify`) {
+          const message = await verifyCommand()
```

Comment:
Returning a non-zero exit code in the case of an error would mean this could be run as part of CI checks

Reply:
I'm taking care of this, thank you 👍